src/elements/dialog/dialog/dialog.stories.ts

```diff
@@ -406,7 +406,7 @@ const StepperTemplate = ({
   <sbb-dialog id="stepper" ${sbbSpread(args)}>
     ${dialogTitle(level, backButton, hideOnScroll, accessibilityCloseLabel, accessibilityBackLabel)}
     <sbb-dialog-content>
-      <sbb-stepper linear orientation="horizontal" size="m">
+      <sbb-stepper orientation="horizontal" size="m">
```

Comment:
Should we re-add?

Reply:
I added parameters in the story for both linear and orientation, in order to test also the animation in vertical mode